### ISSUE TITLE
Add new Mode function for acces only

### ISSUE THREAD

**Issue by seakintruth** [Temtesb/StatisticsCalculationsForExcel#9]
This all should return the mode of a set of data
SELECT TOP 1 Table1.Data
FROM Table1
GROUP BY Table1.Data
ORDER BY Count(Table1.Data) DESC;

**Comment by Temtesb** [comment]
Nice! I will be needing that at work this week.   :)

**Comment by seakintruth** [comment]
Of course, this does not return multiple results if there is more than one mode.
